bitswap/client/client.go

```diff
@@ -196,7 +196,7 @@ func New(parent context.Context, network bsnet.BitSwapNetwork, providerFinder Pr
 	if bs.providerFinder != nil && bs.defaultProviderQueryManager {
 		// network can do dialing.
 		pqm, err := rpqm.New(network, bs.providerFinder,
-			rpqm.WithMaxInProcessRequests(16),
+			rpqm.WithMaxInProcessRequests(8),
```

Comment:
@hsanjuan Do we want exported consts used here?

Should we not set these options if the values provided are the same as the providerquerymanager defaults?